microsoft-edge/toc.yml

```
@@ -1451,6 +1451,12 @@
                 - name: WebView
                   uid: 'WebView2Script!WebView:class'
 
+            - name: HoloLens 2
+              items:
+                - name: Unity Plugin API Reference
+                  href: webview2/reference/HoloLens2/webview2-unity-plugin.md
```

```suggestion
                  href: webview2/reference/hololens2/webview2-unity-plugin.md
```
part of lowercasing dir name

---

done locally, can close